dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasableResourceInterface.java

```diff
@@ -52,8 +51,6 @@
 
     default T updateAliases(User user, Long id, String aliases, String emptyBody) {
```

Comment:
Since this method now always adds an alias or aliases, it's name should reflect that -- right now the name reflects its old behavior.

Reply:
changed name to addAlias